packages/babel-plugin-proposal-class-elements/test/fixtures/to-class-from-private-accessor/basic/output.js

```diff
@@ -0,0 +1,29 @@
+var _privateField = /*#__PURE__*/new WeakMap(),
+    _privateFieldValue = /*#__PURE__*/new WeakSet();
+
+class Cl {
+  constructor() {
+    _privateFieldValue.add(this);
+
+    _privateField.set(this, "top secret string");
```

Comment:
This PR reduces the size of compiled class properties, by storing their values rather than a complete descriptor.

The "old-but-not-legacy" decorator proposal has been rejected because decorators mutating private descriptors where too powerful, so we can be sure that we won't need to support that use case in our implementation.

This is the reason I had to introduce new helpers.